README.md

```diff
@@ -1,17 +1,109 @@
 # Mira S3 Webhosting Sample Application
 
-This application is a very basic application, that provides ready to use project setup for [Mira accelerator].
+This application is a very basic application, that provides a ready to use project setup for the [Mira accelerator].
 
-The successful deployment of this project will create an S3 bucket with web hosting enabled and a single index.html file in it.
+The successful deployment of this project will create an S3 bucket with web hosting enabled and a single index.html file in it, which will output a 'hello world' message.
+
+This sample app is recommended primarily for evaluation purposes, and project owners who want the benefits of Mira CI orchestration but wish to make a clean start with their project.
 
 ## Getting Started
 
-1. Install dependencies with `npm install`
-2. Run `npx mira docs` and navigate to [http://localhost:3000/#/quick-start/](http://localhost:3000/#/quick-start/) for further instructions
+1. Generate your own project repository from this repo [mira-sample-s3-webhosting](https://github.com/nearform/mira-sample-s3-webhosting/generate)
+
+    ⚠️ Remember to make your new repository __🕵🏻‍♀️ Private__ as you store potentially sensitive information in the Mira configuration files.
+
+2. Clone your newly created repository
+
+3. Ensure that you are using Node.js v12:
+
+   ```bash
+   % node --version
+   v12.16.3
+   ```
+
+   or, if using [nvm]:
+
+   ```bash
+   % nvm use
+   Now using node v12.16.3 (npm v6.14.4)
+   ```
+
+4. Install the required dependencies
+
+    Make sure you use a supported version of the CDK (__1.61.1__) and all the required dependencies are included in your package.json file.
```

Comment:
Perhaps add, "Note that the usage of CDK version must be exact.  This is a known constraint of CDK.  Range operators for version will cause your sample to break, e.g. `^1.61.1` is not the same as `1.61.1`.

Reply:
Added